website/docs/d/pi_instance.html.markdown

```diff
@@ -53,7 +53,7 @@ In addition to all argument reference list, you can access the following attribu
   - `network_id` - (String) The network ID of the instance.
   - `network_name` - (String) The network name of the instance.
   - `type` - (String) The type of the network.
-- `deployment_type` - (String) Custom deployment type information (For Internal Use Only).
+- `deployment_type` - (String) Custom deployment type; Allowable value: `EPIC`.
```

Comment:
Can we change this to 
_The custom deployment type._

Reply:
done